department-of-veterans-affairs/va.gov-cms
# VACMS-3445: Remove section fields from checklist paragraph.

## Description
See #3445 

## Testing done
Visual / behat

## Screenshots
N/A

## QA steps
As an administrator, go to `/admin/structure/paragraphs_type/checklist/fields` and visually verify that there is only one field.

### Definition of Done


### Select Team for PR review


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
